README.md

```diff
@@ -12,6 +12,18 @@ Scuttlebutt is a decent(ralised) secure gossip platform.
 >
 > Or, if you're lazy: http://en.wikipedia.org/wiki/Scuttlebutt (laziness will get you nowhere, btw)
 
+## Secure-scuttlebutt vs scuttlebutt vs scuttlebot vs sbot?
```

Comment:
are you sure we want this to be the first thing people see when they browse to scuttlebutt.nz? i'm not so sure.